examples/charts/area.py

```diff
@@ -1,6 +1,7 @@
 from collections import OrderedDict
-from bokeh.charts import Area
-from bokeh.plotting import show, output_file
+
+from bokeh.charts import Area, show, output_file
```

Comment:
so, you want to encourage the use of the transitive import? If I have to choose, I will promote importing from `state`...